xbmc/powermanagement/windows/Win32PowerSyscall.cpp

```diff
@@ -100,7 +157,11 @@ bool CWin32PowerSyscall::CanReboot()
 
 int CWin32PowerSyscall::BatteryLevel()
 {
-  return CWIN32Util::BatteryLevel();
+  int result = 0;
```

Comment:
might as well just `return 0` as this isn't assigned anywhere else and it's such a small function